tests/scenarios/github_test_12/cmd2

```diff
@@ -0,0 +1,7 @@
+source: ubuntu-1-deployment
+cmd: curl google.com
+result: failed
+---
+operation: Network
+condition: SOCK_DGRAM
+action: Block
```

Comment:
the policy action is "Allow". why is it "Block"?

Reply:
The policy action is allow but default posture is Block, this will match the default Posture.